packages/modules/order/src/services/order-module-service.ts

```diff
@@ -2028,7 +2089,7 @@ export default class OrderModuleService<
 
     if (Object.keys(addedShippingMethods).length > 0) {
       const addedShippingDetails = await this.listShippingMethods(
-        { id: Object.keys(addedShippingMethods) },
+        { id: Object.keys(addedShippingMethods), deleted_at: null },
```

Comment:
question: Don't we fetch non-deleted rows by default? If not, should we not?

Reply:
Yes, this is a left over from tests because in some entities the default filter of deleted entities were conflicting with my custom repository and causing an infinity loop in mikro-orm.
I will remove that.